src/covid_model_seiir_pipeline/forecasting/data.py

```diff
@@ -107,50 +107,49 @@ def check_covariates(self, scenarios: Dict[str, ScenarioSpecification]) -> List[
                     raise FileNotFoundError(f'No {covariate_version} file found for covariate {covariate}.')
         return list(regression_covariates)
 
-    def get_infectionator_metadata(self):
```

Comment:
Wait, I thought elastispliner and infectionator were two completely different tools? What's happening here?

Reply:
They are currently.  Methodology is changing and we're trying to get infection space outputs directly out of the elastispliner rather than just deaths.